javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,10 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var newValues = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number' || typeof values[i] === 'object') {
```

Comment:
You can also just check to see if it's not equal to a string. So something like this `if (typeof values[i] !== 'string')`